test/e2e/config/azure-dev.yaml

```diff
@@ -124,16 +124,10 @@ providers:
         targetName: "cluster-template-conformance-presubmit-artifacts.yaml"
       - sourcePath: "${PWD}/templates/test/dev/cluster-template-custom-builds.yaml"
         targetName: "cluster-template-conformance-presubmit-artifacts-windows-containerd.yaml"
-      - sourcePath: "${PWD}/templates/test/ci/cluster-template-prow-external-cloud-provider.yaml"
-        targetName: "cluster-template-external-cloud-provider.yaml"
-      - sourcePath: "${PWD}/templates/test/ci/cluster-template-prow-external-cloud-provider-vmss-flex.yaml"
-        targetName: "cluster-template-external-cloud-provider-vmss-flex.yaml"
-      - sourcePath: "${PWD}/templates/test/ci/cluster-template-prow-azurediskcsi-migration-off.yaml"
```

Comment:
is this cleanup?

Reply:
yes and no, some are cleanup that are no longer used anywhere (like `cluster-template-prow-azurediskcsi-migration-off.yaml` which should have been removed in #3089), some are just file renames to remove the `"-external-cloud-provider"` part